run_local.sh

```diff
@@ -1,7 +1,14 @@
 #!/bin/bash
+
+if [ -z ${SECRET_KEY} ]; then
+    echo "SECRET_KEY is unset."
+    exit 1
+fi
```

Comment:
Can we add `SECRET_KEY` to Docker Compose, rather than requiring the user to set it?